### ISSUE TITLE
Outlook account - Invalid credentials

### ISSUE THREAD

**Issue by brianmoyano** [nylas/N1#532]
Hi, i'm running version 0.3.26-c1ce330 (0.3.26-c1ce330) and i can't seem to add my outlook account.

I fill everything and it says 'Invalid credentials'.

If it helps, i'm using an @live.com account, and if i remember right, i think i could sign in in an earlier version of n1.

Screenshot:
https://www.dropbox.com/s/hw9rmw5f91q6ype/Captura%20de%20pantalla%202015-12-01%2017.13.49.png?dl=0

Thanks.

**Comment by Daktyl198** [comment]
I'm having this same issue using the whatever version is available to download on their site, but I'm using an @outlook.com address

**Comment by burton158** [comment]
Same here. Thought it would be fixed with updates but tried 3 or 4 differents already.
Offtopic: You know if you can change the mails organization to not conversation view? Not so many options so far...

**Comment by jonas-maj** [comment]
@burton158's solution worked for me.

**Comment by brianmoyano** [comment]
Not for me. I only have correct authentications in my activity log.
PS: Updated N1 to it's latest version, the bug persists.

**Comment by rista404** [comment]
Same here, fixed with help from @burton158. Thanks!

**Comment by Daktyl198** [comment]
I can log into my account after using @burton158's solution, but N1 refuses to sync any of my email. I see all of my folders (including the hidden ones like calendar events) but not a single email gets synced. Can't tell if related to this problem or something seperate

**Comment by greensn0w** [comment]
Same problem. 

@burton158 You Rock! :+1: 
Thanks for that awesome hint!

**Comment by JhoanDoe** [comment]
@burton158 thanks a lot, now i can add my outlook accounts.

**Comment by burton158** [comment]
Really glad it works for everybody. For once i can help and not ask for help!

**Comment by mikerockett** [comment]
I have this problem, too. Just downloaded the latest version, and the bug persists. Using outlook.com.

**Comment by kav-ya** [comment]
@brianmoyano, @mikerockett - that's no good.
Could you please email support@nylas.com with the email address you're attempting to connect? (Since this is personal information, we ask that you email us rather than posting here). We'll take a look.

**Comment by mikerockett** [comment]
Hi @kav-ya - I have decided to switch over to another email client, but may look into this again in the future. If I have problems then, I will let you know. Thanks.

**Comment by Jemendieta** [comment]
same problem...help....

**Comment by sittingbool** [comment]
I have the same issue with Version 0.3.36-e8239a9 and an account at hosteurope. The same account works on Apple Mail, Thunderbird, iOS and Android.

**Comment by bluenex** [comment]
Even though this is closed, it seems there is still people who felt into this issue. I have tried @burton158 solution but it didn't work. Finally I came up with **App Password** solution so I'm sharing here just in case.

- go to https://account.microsoft.com/privacy > More security settings > Create a new app password
- use that password to login instead of your actual password

This works for me and hope it would work with others as well.
